field/src/polynomial/mod.rs

```diff
@@ -57,6 +57,11 @@ impl<F: Field> PolynomialValues<F> {
         fft_with_options(coeffs, Some(rate_bits), None)
     }
 
+    pub fn coset_lde(self, rate_bits: usize) -> Self {
```

Comment:
I think the name is a little unclear / possibly misleading; when I saw the calls I initially thought this would treat `self` as poly values over a coset.

Maybe call it something like `lde_onto_coset`? And/or just comment to clarify that the input is treated as points over a subgroup, while the output is treated as points over a coset.

Reply:
Good point, I changed the name and added a comment.